out/index.d.ts

```diff
@@ -1,5 +1,6 @@
 import * as express from 'express';
```

Comment:
Why are these `out` files committed?  They should be generated on demand as part of a `prepare` script - as it stands right now the contents of these files is what actually matters/are the source of truth and any changes in the `ts` version can be ignored if someone forgets to regenerate the files (it also makes conflicts much more likely/harder to resolve if you're committing "binary" files like this)

Reply:
This is true and was one of those annoying things I just kept kicking down the road. I'll open a separate PR to fix this situation.